fe-ionic/init.sh

```diff
@@ -18,11 +18,56 @@ while [[ "$#" > 0 ]]; do case $1 in
 esac; shift; done
 
 echo "generate project"
+<<<<<<< HEAD
```

Comment:
@ManuelFeller  - merge conflict? ....

Reply:
checking right now, had solved that already locally (I thought)

Reply:
OK, no longer in there now, added some explanations about what is happening